lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2012,13 +2012,15 @@ ProcessModule(ModuleSP module_sp, std::string m_description,
           // framework_offset now points to the '/';
 
           std::string parent_path = module_path.substr(0, framework_offset);
-
-          // Never add framework paths pointing into the
-          // system. These modules must be imported from the
-          // SDK instead.
-          if (!StringRef(parent_path).startswith("/System/Library") &&
-              !IsDeviceSupport(parent_path.c_str())) {
-            LOG_PRINTF(GetLog(LLDBLog::Types), "adding framework path \"%s\".",
+          StringRef p(parent_path);
+
+          // Never add framework paths pointing into the system. These
+          // modules must be imported from the SDK instead.
+          if (!(p.startswith("/System/Library") && !IsDeviceSupport(p)) &&
```

Comment:
is the `!(... && !...)` intentional? I am thinking no.

```suggestion
          if (!p.startswith("/System/Library") && !IsDeviceSupport(p) &&
```

Reply:
Oh right!